src/accountsdb/snapshots.zig

```diff
@@ -146,6 +315,28 @@ pub const RentCollector = struct {
     epoch_schedule: EpochSchedule,
     slots_per_year: f64,
     rent: Rent,
+
+    pub fn random(
+        rand: std.Random,
+        params: struct {
+            rent: struct {
+                slots_per_year: f64,
```

Comment:
any reason why we dont randomize this too? should add a comment explaning why

Reply:
I guess there's actually no reason to not randomize this for now - I predicted that it had to be in sync with the `slots_per_year` field in the BankFields, but I don't actually know that to be true.

Reply:
3959c8a60aa75dc4434628200e944608a9207e63